src/test/java/org/cactoos/io/TempFileTest.java

```diff
@@ -106,10 +109,10 @@ public void createFileWithSuffix() throws Exception {
             "randomSuffix%s", System.currentTimeMillis()
         ).asString();
         try (TempFile file = new TempFile("", suffix)) {
-            MatcherAssert.assertThat(
+            new Assertion<>(
                 "File not created with the given suffix",
-                file.value().getFileName().toString(),
-                Matchers.endsWith(suffix)
+                new TextOf(file.value().getFileName().toString()),
+                new StartsWith(suffix)
```

Comment:
@marceloamadeu this test shouldn't pass, you changed from endsWith to StartsWith... if it passes it means the test is broken. It is because you forgot to add `affirm()` on the `Assertion` :)